src/framework/components/light/component.js

```diff
@@ -64,7 +64,7 @@ const _lightPropsDefault = [];
  *
  * Defaults to pc.LIGHTSHAPE_PUNCTUAL.
  * @property {boolean} affectSpecularity If enabled and the light type is pc.LIGHTTYPE_DIRECTIONAL, material specularity
- * will not be affected by this light. Defaults to true.
+ * will be affected by this light. Defaults to true.
```

Comment:
it makes it sounds as if specularity works only for directional light. Perhaps:

`If enabled, material specularity will be affected by this light. Ignored for lights other than {@link LIGHTTYPE_DIRECTIONAL}. Defaults to true.`